Orange/widgets/data/owcolor.py

```diff
@@ -298,12 +298,16 @@ def create_part(variables):
 
     def storeSpecificSettings(self):
         # Store the colors that were changed -- but not others
-        self.current_context.disc_data = \
-            [(var.name, var.values, "colors" in var.attributes and var.colors)
-             for var in self.disc_colors]
-        self.current_context.cont_data = \
-            [(var.name, "colors" in var.attributes and var.colors)
-             for var in self.cont_colors]
+        self.closeContext()
```

Comment:
`storeSpecificSettings()` is called from from within `closeContext()` so this causes an infinite recursion.